code/modules/client/preferences_savefile.dm

```diff
@@ -143,20 +147,21 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 
 /datum/preferences/proc/load_preferences()
 	if(!path)
-		return 0
+		return FALSE
 	if(!fexists(path))
-		return 0
+		return FALSE
 
 	var/savefile/S = new /savefile(path)
 	if(!S)
-		return 0
+		return FALSE
 	S.cd = "/"
 
 	var/needs_update = savefile_needs_update(S)
 	if(needs_update == -2)		//fatal, can't load any data
-		return 0
+		return FALSE
 
 	//general preferences
+<<<<<<< HEAD
 	S["asaycolor"]			>> asaycolor
```

Comment:
```suggestion
	READ_FILE(S["asaycolor"], asaycolor)
```
this and all lines bellow should use `READ_FILES` macro. Not sure why bot didn't mirror that too.